lib/appsignal/cli/diagnose.rb

```diff
@@ -497,10 +496,22 @@ def print_environment(config)
         end
 
         def print_config_options(config)
-          config.config_hash.each do |key, value|
+          # We add the nullified "environment" key to print it ordered
+          # instead of adding it at the top of the list.
+          ordered_config_options = config
+            .config_hash
+            .merge(:environment => nil)
+            .sort
+
+          ordered_config_options.each do |key, value|
             option_sources = sources_for_option(key)
             sources_label = config_sources_label(key, option_sources)
-            puts "  #{key}: #{format_config_option(value)}#{sources_label}"
+
+            if key == :environment
+              print_environment(config)
```

Comment:
Creative solution! 😅 
I've created an issue for the differences in this in the integrations guide: https://github.com/appsignal/integration-guide/issues/94